.github/workflows/script.py

```diff
@@ -0,0 +1,123 @@
+import os
+import yaml
+import sys
+
+os.makedirs('plugins/ImageStream', exist_ok=True)
+
+if os.path.exists("index.yaml"):
+    file = open("index.yaml","r")
+    not_present = 1
+    index = yaml.safe_load(file)
+    for plugin in index['plugins']:
+        if plugin['name'] == 'ImageStreamPlugin':
+            not_present = 0
+            break
+    if not_present:
```

Comment:
You can ignore the suggestion, but I do think you should use something like `present` or `is_present`. Inverting the boolean just gets confusing.

```suggestion
    index = yaml.safe_load(file)
    present = next((plugin for plugin in index['plugins'] if plugin["name"] == "ImageStreamPlugin"), False)
    
    if not present:
```